src/command_modules/azure-cli-ml/azure/cli/command_modules/ml/service/_realtimeutilities.py

```diff
@@ -40,6 +45,8 @@ def upload_dependency(context, dependency, verbose):
        1, 'blob': Success, dependency was a directory, uploaded to blob.
     """
 
+    if dependency.startswith('http') or dependency.startswith('wasb'):
```

Comment:
can you please update the documentation for this method to also reflect that we are returning more than a status code